sim/validator.py

```diff
@@ -116,6 +116,34 @@ def pipeline(
         shutil.rmtree(btb_seq_backup_path)
         shutil.rmtree(results_path)
 
+def benchmark(genomes_path, results_path, output_path):
+    # Validate input
+    if not os.path.exists(results_path):
+        raise Exception("Results path does not exist: ", results_path)
+    if not os.path.exists(genomes_path):
```

Comment:
wondering if this `genome.from_directory(` would be a better place for this?

Reply:
agree, done